### PR TITLE
[map] Enable IPCC_50 for countries

### PR DESCRIPTION
**Pull request by chejennifer**
eg
Screen Shot 2021-11-02 at 10 21 44 AM

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Nice!

**Comment by pradh** [comment]
Will these countries (like Brazil) be clickable now in the world map?

**Comment by chejennifer** [comment]
Yes, these countries will now be clickable in the world map!

File: static/js/tools/map/chart_loader.tsx [hunk 1]
```diff
@@ -134,6 +141,46 @@ export function ChartLoader(): JSX.Element {
...
+  const geoJsonFeatures = [];
+  for (const placeDcid of placeDcids) {
+    if (!placeDcid.includes("/") || !placeDcid.includes("_")) {
+      continue;
+    }
+    const latlon = placeDcid.split("/")[1].split("_");
+    const neLat = Number(latlon[0]) + distance / 2;
+    const neLon = Number(latlon[1]) + distance / 2;
+    const placeName = `${latlon[0]}, ${latlon[1]} (${distance} resolution)`;
+    geoJsonFeatures.push({
```
Comment (beets) [inline static/js/tools/map/chart_loader.tsx:161]: Not sure if the drawing libraries handle overflowing 180 near the intl date line. Add a TODO to handle those cases?

File: static/js/tools/map/chart_loader.tsx [hunk 2]
```diff
@@ -134,6 +141,46 @@ export function ChartLoader(): JSX.Element {
...
+  placeDcids: string[],
+  enclosedPlaceType: string
+): GeoJsonFeature[] {
+  const distance = MANUAL_GEOJSON_DISTANCES[enclosedPlaceType];
+  if (!distance) {
+    return [];
+  }
+  const geoJsonFeatures = [];
+  for (const placeDcid of placeDcids) {
+    if (!placeDcid.includes("/") || !placeDcid.includes("_")) {
```
Comment (beets) [inline static/js/tools/map/chart_loader.tsx:154]: I suspect there might be slowness from doing 2 string includes and 2 string splits over all the dcids.

**Review by beets — APPROVED** [review]
nice!

File: static/js/tools/map/chart_loader.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/map/chart_loader.tsx:167]: done

File: static/js/tools/map/chart_loader.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/tools/map/chart_loader.tsx]: done: removed the 2 string includes

**Review by beets — APPROVED** [review]

**Pull request merged**